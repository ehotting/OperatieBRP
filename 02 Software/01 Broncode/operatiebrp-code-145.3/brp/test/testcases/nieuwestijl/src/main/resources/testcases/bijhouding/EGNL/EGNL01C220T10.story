Meta:
@status                 Klaar
@usecase                UCS-BY.HG

Narrative: R1861 Een HGP mag alleen door betrokken gemeenten worden bijgehouden

Scenario: HGP wordt niet door een betrokken gemeente bijgehouden
          LT: EGNL01C220T10

Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-EGNL/EGNL01C220T10-001.xls

Given pas laatste relatie van soort 2 aan tussen persoon 969557449 en persoon 341199801 met relatie id 2000073 en betrokkenheid id 2000074

When voer een bijhouding uit EGNL01C220T10.xml namens partij 'Gemeente BRP 1'

Then heeft het antwoordbericht verwerking Foutief
Then is het antwoordbericht gelijk aan /testcases/bijhouding/EGNL/expected/EGNL01C220T10.xml voor expressie //brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R
