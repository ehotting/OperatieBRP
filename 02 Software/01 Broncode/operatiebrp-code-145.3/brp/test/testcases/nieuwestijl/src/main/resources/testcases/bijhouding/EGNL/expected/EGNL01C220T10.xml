<brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>88409eeb-1aa5-43fc-8614-43055123a165</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Foutief</brp:verwerking>
        <brp:hoogsteMeldingsniveau>Fout</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="CI_gp">
            <brp:regelCode>R1861</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:beeindigingGeregistreerdPartnerschapInNederland brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
    </brp:beeindigingGeregistreerdPartnerschapInNederland>
</brp:bhg_hgpRegistreerHuwelijkGeregistreerdPartnerschap_R>
